install_config/monitoring/configuring-monitoring.adoc

```diff
@@ -28,4 +28,19 @@ The OpenShift Ansible `openshift_cluster_monitoring_operator` role configures an
 |`openshift_cluster_monitoring_operator_alertmanager_config`
 | Configures Alertmanager. See xref:../install_config/cluster_monitoring.adoc#configuring-alertmanager[Configuring Alertmanager]
 
+|`openshift_cluster_monitoring_operator_prometheus_storage_enabled`
+| Enable persistent storage of Prometheus time-series data.
```

Comment:
`Prometheus` -> `Prometheus'`. You forgot the single quote.

Reply:
Fixed